datadog/provider.go

```diff
@@ -124,7 +124,7 @@ func Provider() *schema.Provider {
 				Type:        schema.TypeInt,
 				Optional:    true,
 				DefaultFunc: schema.EnvDefaultFunc("DD_HTTP_CLIENT_RETRY_TIMEOUT", nil),
-				Description: "The HTTP request retry timeout period.",
+				Description: "The HTTP request retry timeout period in nanoseconds. Defaults to 60 seconds.",
```

Comment:
```suggestion
				Description: "The HTTP request retry timeout period in seconds. Defaults to 60 seconds.",
```

Could you also run `make docs` and commit the changes. Thanks!